scripts/components/reportsDashboards/build.sh

```diff
@@ -72,6 +97,12 @@ echo "BUILD MODULES FOR $PLUGIN_NAME"
 (cd ../../OpenSearch-Dashboards && yarn osd bootstrap)
 echo "BUILD RELEASE ZIP FOR $PLUGIN_NAME"
 (cd ../../OpenSearch-Dashboards/plugins/$PLUGIN_FOLDER && yarn plugin_helpers build)
+echo "DOWNLOADING CHROMIUM FOR $PLUGIN_NAME"
+mkdir -p ../../OpenSearch-Dashboards/plugins/$PLUGIN_FOLDER/build/opensearch-dashboards/$PLUGIN_NAME
+curl -sLO "$CHROMIUM_URL"
+echo "PUTTING CHROMIUM INSIDE $PLUGIN_NAME-$VERSION.zip"
+unzip "$CHROMIUM_TARGET" -d ../../OpenSearch-Dashboards/plugins/$PLUGIN_FOLDER/build/opensearch-dashboards/$PLUGIN_NAME
+(cd ../../OpenSearch-Dashboards/plugins/$PLUGIN_FOLDER/build && zip -ur $PLUGIN_NAME-$VERSION.zip opensearch-dashboards)
 echo "COPY $PLUGIN_NAME.zip"
 cp -r ../../OpenSearch-Dashboards/plugins/$PLUGIN_FOLDER/build/$PLUGIN_NAME-$VERSION.zip $OUTPUT/plugins/
-rm -rf ../../OpenSearch-Dashboards/plugins/$PLUGIN_FOLDER
\ No newline at end of file
+rm -rf ../../OpenSearch-Dashboards/plugins/$PLUGIN_FOLDER
```

Comment:
Wonder why git is saying a change is here

Reply:
CR/LF

Reply:
my editor adds a new line to the end of file automatically..